project/src/types/room-offer.ts

```diff
@@ -0,0 +1,34 @@
+export type RoomOffer = {
+  city:{
+    name:string,
```

Comment:
Пробелы не забывай ставить, чтобы исправить весь файл воспользуйся автоформаттером
```suggestion
    name: string,
```